.sass-lint.yml

```diff
@@ -11,5 +11,5 @@ rules:
   shorthand-values: 0
   no-vendor-prefixes: 0
   no-qualifying-elements: 0
-  nesting-depth: 3
+  nesting-depth: 5
```

Comment:
Yeah, there's 5 levels of depth on the new styles. Some interaction with other rules forces a construct like `div { &[dir=rtl] }` and then another level for the narrow menu case like `.narrow { div { &dir[div=rtl] { } } }`